onnxscript/test/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -0,0 +1,253 @@
+    "nn.functional.relu6": nn_ops.aten_relu6,
+    "nn.functional.selu": core_ops.aten_selu,
+}
+
+TESTED_OPS = frozenset(OPINFO_FUNCTION_MAPPING)
+
+EXPECTED_SKIPS_OR_FAILS = (
+    xfail(
+        "nn.functional.elu",
+        dtypes=dtypes_except(torch.float16, torch.float32),
+        reason="ONNX Runtime doesn't support float64 for Elu",
+    ),
+    xfail(
+        "nn.functional.relu6",
+        dtypes=dtypes_except(torch.float16, torch.float32),
+        reason="ONNX Runtime doesn't support float64 for Relu",
+    ),
+    xfail(
+        "nn.functional.selu",
+        dtypes=dtypes_except(torch.float16, torch.float32),
+        reason="ONNX Runtime doesn't support float64 for Selu",
+    ),
+)
+# END OF SECTION TO MODIFY #####################################################
+
+
+OPS_DB = copy.deepcopy(common_methods_invocations.op_db)
+
+
+class TestOutputConsistency(unittest.TestCase):
+    """Test output consistency between exported ONNX models and PyTorch eager mode.
+
+    This is a parameterized test suite.
+    """
+
+    def setUp(self) -> None:
+        torch.manual_seed(42)
+        np.random.seed(42)
+
+    @common_device_type.ops(
+        [info for info in OPS_DB if info.name in TESTED_OPS],
+        allowed_dtypes=SUPPORTED_DTYPES,
+    )
+    @add_decorate_info(
+        OPS_DB,
+        "TestOutputConsistency",
+        "test_output_match",
+        skip_or_xfails=EXPECTED_SKIPS_OR_FAILS,
+    )
+    def test_output_match(self, device: str, dtype: torch.dtype, op):
+        """Base test method for testing each opset, used by instantiate_device_type_tests."""
+        # device is provided by instantiate_device_type_tests, but we only want to run in cpu.
+        assert device == "cpu"
+
+        samples = op.sample_inputs(
+            device,
+            dtype,
+            requires_grad=False,
+        )
+
+        onnx_function = OPINFO_FUNCTION_MAPPING[op.name]
+        scripted_function = onnxscript.script()(onnx_function)
+
+        for (i, cpu_sample) in enumerate(samples):
+            inputs = (cpu_sample.input, *cpu_sample.args)
+            # Provide the repr to subtest because tensors are not serializable in parallel test runs
+            with self.subTest(
+                sample_num=i,
+                inputs=repr(inputs),
+                kwargs=repr(cpu_sample.kwargs),
+            ):
+                input_numpy = [x.numpy() for x in inputs if isinstance(x, torch.Tensor)]
+                torch_output = op(*inputs, **cpu_sample.kwargs)
+                try:
+                    function_output = scripted_function(*input_numpy, **cpu_sample.kwargs)
+                except onnxruntime.capi.onnxruntime_pybind11_state.NotImplemented:
+                    self.skipTest(
+                        f"ONNX Runtime doesn't support running {op.name} with dtype {dtype}",
+                    )
+
+                # Use torch testing to ensure dtypes and shapes match
+                torch.testing.assert_close(
```

Comment:
Do we need to consider the case of multiply outputs?

How about this:

assert [torch.allclose(o, torch.tensor(o_ort)) for o, o_ort in zip(torch_output, function_output)]

Reply:
Good catch. I am inclined to keep it as is for now and expand to multi input when needed